meson_options.txt

```diff
@@ -118,11 +118,6 @@ option('pext',
        value: false,
        description: 'Use the pext instruction')
 
-option('android',
```

Comment:
This is no longer needed as one can detect android on meson cross build with host_machine.system() == 'android'